packages/mobile/src/screens/favorites-screen/FavoritesScreen.tsx

```diff
@@ -32,13 +35,20 @@ const favoritesScreens = [
 
 export const FavoritesScreen = () => {
   usePopToTopOnDrawerOpen()
+  const { isEnabled: isOfflineModeEnabled } = useFeatureFlag(
+    FeatureFlags.OFFLINE_MODE_ENABLED
+  )
 
   return (
     <Screen>
       <Header text='Favorites' />
-      <ScreenContent>
+      {isOfflineModeEnabled ? (
```

Comment:
Maybe a quick comment here that ScreenContent handles displaying the offline indicator. I was confused for a sec